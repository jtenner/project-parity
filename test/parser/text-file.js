const { OptionType } = require("../../lib");

module.exports = {
  argv: ["--option-a", "test/files/a.txt"],
  config: {
    optionA: {
      type: OptionType.TextFile,
    },
  },
};
