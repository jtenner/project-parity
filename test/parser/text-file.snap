{
  args: [ 'test\\parser\\text-file.config.js' ],
  cliProvided: Set(1) { 'optionA' },
  configProvided: Set(0) {},
  errors: [],
  options: { optionA: 'a b c d e f g\r\n' },
  unknown: [ '--config' ]
}
